bin/uglifyjs

```diff
@@ -575,6 +581,15 @@ function read_whole_file(filename, cb) {
     }
 }
 
+function read_source_map(code) {
+    var match = /\n\/\/# sourceMappingURL=data:application\/json(;.*?)?;base64,(.*)/.exec(code);
+    if (!match) {
+        print_error("ERROR: can't read source map");
+        process.exit(1);
```

Comment:
I don't think it should be a fatal error if the upstream code lacks an inline source map.

Perhaps something like "WARNING: inline source map not found" and `return null;` instead of `process.exit(1);`?